PHPCompatibility/Sniffs/FunctionUse/RemovedFunctionParametersSniff.php

```diff
@@ -99,12 +93,24 @@ class RemovedFunctionParametersSniff extends AbstractFunctionCallParameterSniff
                 '8.1'  => false,
             ],
         ],
-        'imagefilledpolygon' => [
+        'imagepolygon' => [
             4 => [
                 'name' => 'num_points',
                 '8.1'  => false,
             ],
         ],
+        'imagerotate' => [
+            4 => [
+                'name' => 'ignore_transparent',
+                '8.3'  => true,
+            ],
+        ],
+        'imap_headerinfo' => [
```

Comment:
I'm confused about this one. Doesn't seem to have anything to do with the imagerotate changes ?

Reply:
The imagerotate change is in the second commit: https://github.com/PHPCompatibility/PHPCompatibility/pull/1619/commits/55e0202e8c085ee381a5d856a5d503b17153079d

The first commit fixes the order of the list.

Reply:
Very odd, the Github view didn't show the removal at the top before. Now it does.

Reply:
The weird ways of the interwebs... 🤷🏻‍♀️